src/js/structs/PodContainer.js

```diff
@@ -58,6 +58,16 @@ module.exports = class PodContainer extends Item {
     return this.get('name') || '';
   }
 
+  getResources() {
+    let resources = this.get('resources') || {};
+    return Object.assign({
+      cpus: 0,
+      mem: 0,
+      gpus: 0,
+      disk: 0
+    }, resources);
```

Comment:
#nit
we could just `Object.assign({...}, this.get('resources'));`
`Object.assign` takes care of `undefined` and `null`